services/copier/internal/internal.pb.go

```diff
@@ -15,14 +15,16 @@ It has these top-level messages:
 package internal
 
 import proto "github.com/gogo/protobuf/proto"
+import fmt "fmt"
```

Comment:
it happens when you run `go generate ./...`  I suppose I could just scope it to the file I changed, but it is going to get this same change when it builds.